packages/ts-transformers/src/internal/decorators/property.ts

```diff
@@ -58,11 +59,41 @@ export class PropertyVisitor implements MemberDecoratorVisitor {
     }
     const options = this._augmentOptions(arg0);
     const name = property.name.text;
-    litClassContext.litFileContext.nodeReplacements.set(property, undefined);
+    const factory = this._factory;
+
+    if (isGetter && property.decorators) {
+      // Filter out the current decorator
+      let decorators: ts.Decorator[] | undefined = property.decorators.filter(
+        (dec) => dec !== decorator
+      );
+
+      // If there are no decorators prevent the tslib package from being
+      // imported by unassigning the decorators array.
+      if (decorators.length === 0) {
+        decorators = undefined;
+      }
```

Comment:
Can simplify: On line 78 pass `undefined`, and the `decorators` variable and this logic can be removed.

Reply:
do we care if there are other user-defined decorators?

Reply:
Great point! Maybe make that support explicit by adding a test.

I don't think the property declaration supports other user-defined decorators, so maybe not required?